### ISSUE TITLE
Notification plugin doesn't work config.toolbarLocation set to 'bottom'

### ISSUE THREAD

**Issue by andrewpurkett** [ckeditor/ckeditor4#624]
## Are you reporting a feature or a bug?
Bug

## Provide detailed reproduction steps (if any)
1. Use the following editor configuration with full package editor:
```
CKEDITOR.replace( 'editor1', {
	extraPlugins : 'notification',
	toolbarLocation: 'bottom'
} );
```
2.  Open browser developer-tools and switch to console tab
3. Press Paste From Word button to show notification (The easiest but not the only way to reproduce the problem).

### Expected result

(In Chrome, Firefox, etc.) "Your browser doesn't allow you to paste this way..." notification displayed within editor and no error is thrown.

### Actual result

Notification is shown below the editor and JavaScript error is thrown:

```
Message: top is null
Location: /ckeditor-dev/plugins/notification/plugin.js
Line: 642
Code: 
641: top = editor.ui.space( 'top' );
642: topRect = top.getClientRect();
```
This error subsequently happens every time any toolbar button is pressed.

## Other details

* Browser: Any
* OS: Any
* CKEditor version: 4.7.1
* Installed CKEditor plugins: notification, toolbar

@andrewpurkett  I have updated your ticket description so that it is more generic and doesn't give the influence that Paste From Word has something to do with this error.

**Comment by wimleers** [comment]
I'm seeing this with CKEditor 4.7.1 even *without* configuring `toolbarLocation`, in Drupal `8.4.0-beta1`.

**Comment by jswiderski** [comment]
This problem is not only related to Chrome.It cab be reproduced in any browser and all you need as actually `toolbarLocation` set to `bottom` and Notification plugin:  https://codepen.io/j_swiderski/pen/wvajPzG.

**Comment by jswiderski** [comment]
NOTE:
Johan De Clercq from Oracle noticed that changing line https://github.com/ckeditor/ckeditor4/blob/major/plugins/notification/plugin.js#L641  from
```
top = editor.ui.space( 'top' );
```
to 
```
top = editor.ui.space( editor.config.toolbarLocation ); 
```
causes the error no longer being thrown. I'm not sure if there are any other issues, however, this is something worth checking.

**Comment by f1ames** [comment]
Fixed with c29775a.
